freqtrade/optimize/backtesting.py

```diff
@@ -464,13 +489,13 @@ def _enter_trade(self, pair: str, row: Tuple, stake_amount: Optional[float] = No
                      trade: Optional[LocalTrade] = None) -> Optional[LocalTrade]:
 
         # let's call the custom entry price, using the open price as default price
-        propose_rate = strategy_safe_wrapper(self.strategy.custom_entry_price,
-                                             default_retval=row[OPEN_IDX])(
-            pair=pair, current_time=row[DATE_IDX].to_pydatetime(),
-            proposed_rate=row[OPEN_IDX])  # default value is the open rate
-
-        # Move rate to within the candle's low/high rate
-        propose_rate = min(max(propose_rate, row[LOW_IDX]), row[HIGH_IDX])
+        order_type = self.strategy.order_types['buy']
+        propose_rate = row[OPEN_IDX]
+        if order_type == 'limit':
```

Comment:
This might be something we can add to the live version, too ...

Reply:
I think it would make sense to do same in live. I blindly assumed that was the case already. Makes little sense to ask strategy for entry price and then not use it